Foundation/NSNumber.swift

```diff
@@ -452,6 +456,12 @@ public class NSNumber : NSValue {
     public func compare(otherNumber: NSNumber) -> NSComparisonResult {
         return ._fromCF(CFNumberCompare(_cfObject, otherNumber._cfObject, nil))
     }
+
+    public func descriptionWithLocale(locale: AnyObject?) -> String {
+        guard let aLocale = locale else { return description }
```

Comment:
from a subclassing perspective, iirc this is a bit inverted - descriptionWithLocale should be the funnel point and description should call descriptionWithLocale(nil) but that is a minor 'nit since this is pretty reasonable as it stands.